src/Views/Admin/Pages/templates/setup-template.php

```diff
@@ -0,0 +1,94 @@
+<div class="wrap" class="give-setup-page">
+
+	<h1 class="wp-heading-inline">
+		<?php echo __( 'Setup GiveWP', 'give' ); ?>
```

Comment:
@kakshak always uses `esc_html__` if there is the only in the translateable string.

Reply:
fixed